File: 0069-sqrtx/0069-sqrtx.java
class Solution {
    public int mySqrt(int x) {
        if(x>=2147395600){
            return 46340;
        }else{
            int signal = -1;
        int counter1 = 0;
        int counter2 = 1;
        int result = -1;
        while(signal<0){
            int a = counter1*counter1;
            int b = counter2*counter2;
            if(a<=x && x<b){
                result = counter1;
                signal = 1;
            }
            counter1++;
            counter2++;
        }

        return result;
        }
        
    }
}

File: 0069-sqrtx/0069-sqrtx.cpp
class Solution {
public:
    int mySqrt(int x) {
        if(x == 2147483647){
            return 46340;
        }
        int root = 0;
        while(true){
            int square = root*root;
            if(square == x){
                return root;
            }else if(square>x){
                return root - 1;
            }
            root++;
        }
        return 0;
    }
};